lib/testing/src/lib/content-services/rest-apis/upload/upload-api.ts

```diff
@@ -0,0 +1,68 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RepoApi } from '../repo-api';
+// import { E2E_ROOT_PATH } from '../../../../configs';
+import { UploadApi as AdfUploadApi } from '@alfresco/js-api';
+
+// const fs = require('fs');
+
+export class UploadApi extends RepoApi {
+  upload = new AdfUploadApi(this.alfrescoJsApi);
+
+  constructor(username?: string, password?: string) {
+    super(username, password);
+  }
+
+//   async uploadFile(fileName: string, parentFolderId: string = '-my-') {
```

Comment:
do we need this part?

Reply:
yes, it's needed as well, I had it commented out cause I needed to talk with @rdiacenco  and @cristinaj  about how to proceed with this one, since this has parts that overlap with already existing upload.actions.
we will keep both in parallel for now, until we can change the test files to use this new upload-api, then we can remove upload.actions.

Reply:
my next commit will add the upload-api as well